docs/cli.rst

```diff
@@ -0,0 +1,853 @@
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -with-tower               |            | Monitor workflow execution with Seqera Tower service.                          |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -with-trace               |            | Create processes execution tracing file.                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -with-weblog              |            | Send workflow status messages via HTTP to target URL.                          |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -without-docker           | false      | Disable process execution with Docker.                                         |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -without-podman           |            | Disable process execution in a Podman container.                               |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -w, -work-dir             |            | Directory where intermediate result files are stored.                          |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
+
+
+--------------------
+list          
+--------------------
+
+
+Description
+^^^^^^^^^^^^^^^^^^^^
+List all downloaded projects
+
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+The ``list`` command can be invoked like so::
+
+
+    $ nextflow list [options]
+
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+
+
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -help, -h                 |  false     | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
+
+
+--------------------
+log           
+--------------------
+
+
+Description
+^^^^^^^^^^^^^^^^^^^^
+Print executions log and runtime info
+
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+
+The ``log`` command can be invoked like so::
```

Comment:
Could you please guide me on the usage of this command - I've tried a few permutations and I only seem to generate the same result.

A couple of examples here would be great 🆘

Reply:
See 11.4. Execution provenance at https://seqera.io/training/.

Reply:
Gotcha!